pkg/rancher-desktop/components/StatusBarItem.vue

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+import Vue, { PropType, VueConstructor } from 'vue';
+
+import NetworkStatus from '@pkg/components/NetworkStatus.vue';
+import Version from '@pkg/components/Version.vue';
+
+export default Vue.extend({
+  props: {
+    data: {
+      type:    Object as PropType<{ label: string, value: string }>,
+      default: null,
+    },
+    subComponent: {
+      type:    String,
+      default: null,
+    },
+    icon: {
+      type:     String,
+      required: true,
+    },
+  },
+  computed: {
+    getSubComponent(): VueConstructor | undefined {
+      if (this.subComponent) {
+        return this.subComponent === 'Version' ? Version : NetworkStatus;
+      }
+
+      return undefined;
+    },
+    getTooltip(): { content: string, placement: string, classes: string } {
+      return {
+        content:   `<b>${ this.t(this.data.label) }</b>: ${ this.data.value }`,
+        placement: 'top',
+        classes:   'tooltip-footer',
+      };
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="status-bar-item">
```

Comment:
The mouse cursor should be default instead of the I-beam for each status item. I think that we should also disable selection.

Reply:
I'll change the good accordingly. Although I'm not 100% sure about it. I can imagine it's neat to be able to copy e.g. the Rancher Desktop version for a bug report. Maybe we can discuss further options at some point. The change is trivial and easy to revert/adapt if we decide.

Reply:
I think you're right, we should leave the cursor as-is and allow for selection in the status bar. I think that supporting users so that they can quickly copy is enough of a reason to keep things as-is. 

We can follow-up to see if we want to implement some sort of "click to copy" mechanism, but that would definitely be an additional requirement if we choose to go that direction.